src/Internal/QueryM.purs

```diff
@@ -49,6 +44,7 @@ module Ctl.Internal.QueryM
 
 import Prelude
 
+-- import Ctl.Internal.QueryM.Kupo (isTxConfirmed)
```

Comment:
Can we remove this?

Reply:
Yeah, it was just a leftover from trying to import isTxConfirmed directly - it caused import cycles, so I just went with passing the function as an arg, similar to how we did with ODC's ws already